src/checkers/inference/InferenceDevelLauncher.java

```diff
@@ -98,7 +98,7 @@ public String getInferenceCompilationBootclassPath() {
 
         final List<String> newPathOpts = new ArrayList<String>();
 
-        if (!cp.trim().isEmpty()) {
+        if (cp != null && !cp.trim().isEmpty()) {
```

Comment:
Why allow null? Doesn't that mean that the expected property wasn't set and something is wrong? I would either error with a nice message or explain why this is ok.

Reply:
You are right. I somehow inherit this change during playing with the classpath and bootclasspath. It's a trial change and should be removed.

Will add a nullness check on `cp`.